templates/index.html

```diff
@@ -2,17 +2,49 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+    <div class="jumbotron">
+        <h1 class="display-4">To-Do App</h1>
+        <p class="lead">Just another to-do app.</p>
+    </div>
+
+    <div class="row">
+        <div class="col-md-12">
+            <h2>Items</h2>
+            <ul class="list-group">
+                {% for item in items %}
+                    {% if item.status == 'Not Started' %}
+                        <li class="list-group-item">{{ item.title }} <br> {{ item.desc }}
+                            <span class="badge badge-danger" style="float: right">
+                                <form method="post" action="{{ url_for('complete_item', id =item.id) }}">
+                                    <button class="btn btn-link" type="submit" data-toggle="tooltip"
+                                            title="Mark as completed">
+                                    </button>
+                                </form>{{ item.status }}
```

Comment:
I think you meant to put item.status inside the `<button>...</button>`
Currently the button doesn't have any text so it's not obvious it's there and clicking on the text does nothing.